Pear/Controller/SchedulingTimeViewController.swift

```diff
@@ -519,6 +532,87 @@ class SchedulingTimeViewController: UIViewController {
             self.errorMessageAlertView.transform = .identity
         })
     }
+    
+    private func showEmailMessageAlertView() {
+        
+        emailMessageAlertView = MessageAlertView(
+            mainMessage: "Send your Pear a personal note.",
+            actionMessage: "Send an email",
+            dismissMessage: "Skip",
+            alertImageName: "happyPear",
+            removeFunction: { [weak self] isDismiss in
+                guard let self = self else { return }
+                guard let match = self.match else { return }
+                if !isDismiss {
+                    let matchEmail = "\(match.pair ?? "")@cornell.edu"
```

Comment:
Do we want to just show `@cornell.edu` if there is no email? Wouldn't it mess things up if you tried to open that address?